### DISCUSSION TITLE
add Multiplayer saves to Steam Cloud

### DISCUSSION THREAD

**Discussion by James-E-A** [FakeFishGames/Barotrauma#9050] [Ideas and suggestions]
Steam Cloud only backs up my _singleplayer_ campaign saves; multiplayer campaign saves and WIP local modifications to subs are excluded.

* If a SP campaign save is lost due to PC failure, it affects _only_ me.
* If an MP campaign is lost due to PC failure, if affects everyone on the campaign.
* If a WIP local sub is lost due to PC failure, if affects everyone who was looking forward to the changes to that sub.

It seems a bit strange to _only_ back up the gamefiles that have the _least_ impact if lost.

Why is the `Multiplayer` subdirectory excluded from the Steam Cloud backups?

**Comment by Deliphin** [comment]
I just want to say I just lost my group's save because this isn't implemented yet. Ugh.
I can't think of a single reason why you'd *not* want your multiplayer saves on the cloud.
